manifests/candlepin.pp

```diff
@@ -130,6 +121,30 @@
       key_mode    => '0440',
     }
 
+    file { $keystore_password_path:
+      ensure  => file,
+      content => $keystore_password,
+      owner   => 'root',
+      group   => $group,
+      mode    => '0440',
+    }
+
+    keystore { $keystore:
+      ensure        => present,
+      password_file => $keystore_password_path,
+      owner         => 'root',
+      group         => $group,
+      mode          => '0640',
+    }
+
+    keystore_certificate { "${keystore}:tomcat":
+      ensure        => present,
+      password_file => $keystore_password_path,
+      certificate   => $tomcat_cert,
+      private_key   => $tomcat_key,
+      ca_file       => $ca_cert,
```

Comment:
should *all* these be called `*_file`, as they are all… files?

Reply:
I have seen in other modules related to certs not attaching `_file` so I lean towards going with that model and it feels tidy.